src/app/components/TextField/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const TextField = forwardRef(({ title, inputVariant, error, ...rest }, ref) => (
+  <div className={styles['textfield-container']}>
+    <label className={styles.title}> {title} </label>
+    <input
+      ref={ref}
+      className={`full-width ${!!error && 'input-error'} ${styles['text-field']} ${inputVariant}`}
```

Comment:
same, call scss class with camelCase, and use classnames for conditional classes